tests/wallet/db_wallet/test_dl_wallet.py

```diff
@@ -239,6 +239,9 @@ async def is_singleton_confirmed(lid: bytes32) -> bool:
 
         new_root = MerkleTree([Program.to("root").get_tree_hash()]).calculate_root()
         dl_tx = await dl_wallet.create_update_state_spend(launcher_id, new_root)
+        with pytest.raises(ValueError) as exception:
+            await dl_wallet.create_update_state_spend(launcher_id, new_root)
+            assert "is currently pending" in str(exception.value)
```

Comment:
I'm pretty sure the assertion does nothing at present since the exception would be raised out of the `await` and flow then jumps to the context manager exit.  The assert would have to be dedented.  But, `match=` should work.

```suggestion
        with pytest.raises(ValueError, match="is currently pending"):
	    await dl_wallet.create_update_state_spend(launcher_id, new_root)
```